project_pivot/views/project_project.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+    Copyright 2024 Tecnativa - Carolina Fernandez
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <record id="project_pivot_view" model="ir.ui.view">
+        <field name="name">project.project.pivot</field>
+        <field name="model">project.project</field>
+        <field name="arch" type="xml">
+            <pivot>
+                <field name="name" type="row" />
+                <field name="date_start" type="col" />
+            </pivot>
+        </field>
+    </record>
+        <!-- As stated in the core -->
+        <!-- Please update both act_window when modifying one (open_view_project_all or open_view_project_all_group_stage) -->
+        <!-- as one or the other is used in the menu menu_project_config -->
+        <record id="open_view_project_all_pivot" model="ir.actions.act_window.view">
+            <field name="act_window_id" ref="project.open_view_project_all" />
+            <field name="view_mode">pivot</field>
+            <field name="sequence">4</field>
+            <field name="view_id" ref="project_pivot_view" />
+        </record>
+
+        <record
+        id="open_view_project_all_group_stage_pivot"
+        model="ir.actions.act_window.view"
+    >
+            <field
+            name="act_window_id"
+            ref="project.open_view_project_all_group_stage"
+        />
+            <field name="view_mode">pivot</field>
+            <field name="sequence">5</field>
+            <field name="view_id" ref="project_pivot_view" />
+        </record>
+        <record id="project.open_view_project_all" model="ir.actions.act_window">
+            <field name="view_mode">kanban,tree,form,pivot</field>
+        </record>
+        <record
+        id="project.open_view_project_all_group_stage"
+        model="ir.actions.act_window"
+    >
+           <field name="view_mode">kanban,tree,form,calendar,pivot,activity</field>
```

Comment:
By default, it should show the Kanban view, but the List view is currently the first view displayed. Can you check this?
![image](https://github.com/user-attachments/assets/2db7b8d3-6032-4487-a795-f7859d65ce31)


Before this PR:
![image](https://github.com/user-attachments/assets/9aa4869b-82eb-43c7-941c-c4a62f74e4f9)

Reply:
The problem is with project_list. https://github.com/OCA/project/blob/16.0/project_list/views/project_project.xml#L41
What should we do @pedrobaeza ?